_pages/how-we-work/purchase-requests.md

```diff
@@ -11,26 +11,7 @@ Some types of purchases are handled differently. See these pages for guidance ab
-
-Please note that for training and conferences, someone from the TTS Ops team will enter the request in C2 on behalf of the requestor. Please be sure to follow the steps in the [training section](https://handbook.18f.gov/conferences-events-training/#im-attending-training-that-is-not-at-a-conference) or [conferences section](https://handbook.18f.gov/conferences-events-training/#im-attending-a-conference-including-training-offered-before-or-after-the-conference).
-
-Here's the information you'll need to enter in C2:
-
-* **Purchase Type:** Software/Services/Etc.
-* **Product Name and Description:** Include the specifics of what you are requesting.
-* **Justification:** The justification section is very important. This is how you will get approval from TTS for what you need. It needs to be a strong justification for why you need it and you must include how it is different from something we have already procured and is available for use.
-* **Link to Product:** Provide a link to the product website, if available.
-* **Cost per unit:** If the product costs less than $3,500 per unit, then you must enter the unit price (and not the total price) here. If the product costs over $3,500 per unit (still must be under $10,000), then you will need to enter the amount as $3,500 and put the actual amount in the description for now (we are working to update C2 accordingly).
-* **Quantity:** Include the quantity of what you are requesting.
-* **Additional Information:** Include your office/chapter/portfolio and your supervisor’s name.
-
-After the C2 request is created:
-
-* Add your supervisor as an observer in C2 by scrolling all the way to the bottom where it says “Observers” and typing in their name or email. (If they have never used C2, you may need to request access for them, like you did for yourself at the beginning of the process.)
-* Then, supervisor adds a comment “Approved” in your C2 request. Please note that for OPP purchases, the OPP Director’s approval is required in addition to the portfolio supervisor’s approval.
+For all requests, submit the [micropurchase request form] (https://docs.google.com/forms/d/e/1FAIpQLSd-GoOE9xWWfJvdZNRP3SE7mj5ysI_RfM8brxdG8YpyJV9yKA/viewform) to initiate the request process. Upon completing it, you will get an automated email, which confirms the details you entered and provides all of the actions that you will need to complete for the request. Your supervisor may want to discuss the request with you before proceeding with further approvals.
```

Comment:
You can use the [micropurchase request form](https://docs.google.com/forms/d/e/1FAIpQLSd-GoOE9xWWfJvdZNRP3SE7mj5ysI_RfM8brxdG8YpyJV9yKA/viewform) to request an official purchase of anything under $10,000. Upon completing it, you will get an automated email, which confirms the details you entered, outlines any further actions you need to take, and loops in everyone who would need to approve your type of request. Different types of requests have different timelines for completion, but you should expect a response from whoever is identified to review your request within two business days.

Reply:
^ suggested edit-- supervisor is required for some, but not all requests. The form clarifies this more.